root/static/scripts/common/components/EntityLink.js

```diff
@@ -218,7 +218,14 @@ const EntityLink = ({
         <Comment className="comment" comment={comment} key="comment" />,
       );
     }
-    if (entity.entityType === 'area') {
+    if (entity.entityType === 'recording') {
+      parts.push(
+        ' ',
+        <span className="comment" key="video">
+          {bracketed(l('video'), {__react: true})}
```

Comment:
I think it should be fine to always show `(video)` now, but let me know if that would be out of place somewhere.

Reply:
Is it about [MBS-9306](https://tickets.metabrainz.org/browse/MBS-9306)?

On one hand, it would be redundant with the video cell/icon in recordings tab for artist page, but on the other, it would add missing info to the recordings (rels) table in work page.

Btw, I spotted another bug about video cell/icon ruining layout in recording search results, e.g. https://musicbrainz.org/search?query=Last+Angel&type=recording

It would be nice to display the video icon everywhere for consistency.

Reply:
I think it's an unrelated issue with the inline search according to the [linked thread](https://community.metabrainz.org/t/track-recording-titles-for-bonus-dvds/244281/15) (I'm also having trouble reproducing it).

Good point about the video icons...should we remove the `(video)` disambiguation everywhere, but display the icon everywhere in its place? Or we can keep `(video)` in the inline search (in addition to the icon), but only show the icon next to links.

Reply:
I pushed some changes to standardize the video recording display, and fix the recording search results.

The changes are described in 231a8e2acb0bf7f063a31e6d173315814dd73b2f. Let me know how that looks.

Reply:
Better! That many bugs fixed at once! :fireworks: 

Assuming the `title` attribute of the video icon `span` tag is used by screen readers, maybe the video icon can replace the (video) comment? That would realign titles and remove redundancy.

Reply:
Removed the (video) comment (agreed it seems redundant now) but I left the video icon on the left side of the links since that's how we display country flags and editor avatars too. Though there might be a good reason to make them different.